src/components/VmDetails/BaseCard.js

```diff
@@ -76,9 +76,7 @@ class BaseCard extends React.Component {
 
   renderChildren (childProps) {
     const children = this.props.children || noop
-    return typeof children === 'function' ? children(childProps)
-      : React.isValidElement(children) ? React.cloneElement(children, childProps)
-        : children
+    return React.isValidElement(children) ? React.cloneElement(children, childProps) : children(childProps)
```

Comment:
React component can be function as well, but not every function is React component. Thus we need to check first React component, because React component can include some props.